adapters/playwire_ortb/playwire_ortb.go

```diff
@@ -0,0 +1,172 @@
+package playwire_ortb
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+// GumGumAdapter implements Bidder interface.
+type GumGumAdapter struct {
+	URI string
+}
```

Comment:
My first thought would be to rename this to something like `PlaywireORTBAdapter`.  That would make it easier to distinguish from the similarly named struct in GumGum's adapter, and it looks like it would follow the convention of the other adapters.  I'm not sure if that would trigger a lot of additional work to rename things further downstream, though.